src/BuildIt.Forms/BuildIt.Forms.Controls/Platforms/Android/CameraPreviewControlRenderer.cs

```diff
@@ -158,14 +159,19 @@ protected override void OnElementChanged(ElementChangedEventArgs<Frame> e)
         }
 
         /// <inheritdoc />
-        protected override void OnElementPropertyChanged(object sender, PropertyChangedEventArgs e)
+        protected override async void OnElementPropertyChanged(object sender, PropertyChangedEventArgs e)
         {
             base.OnElementPropertyChanged(sender, e);
 
+
             if (e.PropertyName == CameraPreviewControl.PreferredCameraProperty.PropertyName)
             {
                 SwitchCameraIfNecessary();
             }
+            else if (e.PropertyName == CameraPreviewControl.EnableContinuousAutoFocusProperty.PropertyName)
```

Comment:
Would be better to use nameof(EnableContinuousAutoFocus) instead of .PropertyName

Reply:
fails though when the name of the property passed to the BindableProperty.Create(...) method doesn't use nameof(...) for whatever reason